content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -510,7 +510,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `RuntimeClass`: Enable the [RuntimeClass](/docs/concepts/containers/runtime-class/) feature for selecting container runtime configurations.
 - `ScheduleDaemonSetPods`: Enable DaemonSet Pods to be scheduled by the default scheduler instead of the DaemonSet controller.
 - `SCTPSupport`: Enables the _SCTP_ `protocol` value in Pod, Service, Endpoints, EndpointSlice, and NetworkPolicy definitions.
-- `ServerSideApply`: Enables the [Sever Side Apply (SSA)](/docs/reference/using-api/api-concepts/#server-side-apply) path at the API Server.
+- `ServerSideApply`: Enables the [Sever Side Apply (SSA)](/docs/reference/using-api/api-concepts/server-side-apply/) path at the API Server.
```

Comment:
@tengqm , Is this linking to the new SSA page or the original SSA heading in 'api-concepts'?

Reply:
That is the new page.

Reply:
The new page is:
`/docs/reference/using-api/server-side-apply/ `

https://deploy-preview-23863--kubernetes-io-master-staging.netlify.app/docs/reference/using-api/server-side-apply/

Reply:
Fixed.